official-antistasi-community/A3-Antistasi
# moved anti keybinds from eh to built in keybind introduced in arma3 v…

…2.06

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    since arma 3 v2.06 and the move to mod, we are now able to have custom keybinds similare to vanilla keybinds of arma, this pr removes the old key eh and replaces them with the new native keybind system.
    per https://community.bistudio.com/wiki/Arma_3:_Modded_Keybinding

### Please specify which Issue this PR Resolves.
closes #1760

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
1) load into mission
2) test keybinds
3) open configure -> controls -> antistasi
and modify the keybinds there
4) test with new keybinds
5) reset keybinds to default
7) check that you cant place vehicles while battleMenu is open (only thing blocked currently)
6) modify keybind, unload mod, launch, the relaunch with mod loaded again, then check keybinds stored properly

********************************************************
Notes:
